src/firefly/java/edu/caltech/ipac/firefly/server/servlets/ServerStatus.java

```diff
@@ -43,8 +48,7 @@
 
     protected void processRequest(HttpServletRequest req, HttpServletResponse res) throws Exception {
 
-        boolean showHeaders = Boolean.valueOf(req.getParameter("headers"));
-        String jwt = req.getParameter("jwt");
+        boolean showHeaders = Boolean.parseBoolean(req.getParameter("headers"));
```

Comment:
This appears to be a special view of the server status page. why not just add it to everything else?

Reply:
Yes.  I added this a while back.   Displaying headers may take up lots of space that's why I did not add it by default.  Also, I am thinking we can expand on this and have additional parameters as needed in the future, like actions.